glide.lock

```diff
@@ -667,4 +728,61 @@ imports:
   - tools/clientcmd/api
   - tools/metrics
   - transport
-testImports: []
+testImports:
```

Comment:
@vdemeester just in case you missed it in the fog of war: I tried this, and did not receive a green. https://travis-ci.org/containous/traefik/jobs/227970015 is the respective job.